puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/gemset_cleanup.sh

```diff
@@ -4,5 +4,6 @@
 . /etc/profile.d/rvm.sh
 gemset=$(echo ${JOB_NAME} | cut -d/ -f1)-${EXECUTOR_NUMBER}
 rvm use ruby-${ruby}@${gemset}
+bundle exec rake db:drop
```

Comment:
Updated to check if foreman/ exists -- https://github.com/theforeman/foreman-infra/pull/195/files#diff-fd2ec544c14fd176a5ae4d65c093af34R4